runtime/onert/core/include/backend/train/ExtraTensor.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_EXTRA_H__
+#define __ONERT_BACKEND_EXTRA_H__
+
+#include <backend/basic/Tensor.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace train
+{
+
+// ExtraTensor is a tensor that is accessed only by one opeartion layer.
+// In other word, Extra tensor's scope is limited into a specific layer.
```

Comment:
Is this sentence understandable? 

Let me know if it is hard to catch the meaning, I could reconsider this sentence if so.

Reply:
```suggestion
// ExtraTensor is a tensor that is accessed only within one operation layer.
// In the other words, the scope within which an Extra tensor can be used is confined to one specific layer."
```

Reply:
Thanks! Updated a bit.